content/reference/enums.md

```diff
@@ -214,38 +214,47 @@ weekend_night                  | Rule applied for weekend night.
 
 ## Rental types
 
+<div class="callout callout-warning" markdown="1">
```

Comment:
Why do we need that info here?

Reply:
because it's not released yet and if someone will use those rental_types for pro accounts it will fail on validation. Also, we need at least 2 weeks notice before releasing it so I assumed it should be stated when exactly it's available